dev/com.ibm.ws.security.spnego_fat/fat/src/com/ibm/ws/security/spnego/fat/BasicAuthTest.java

```diff
@@ -465,6 +465,9 @@ public void testSpnegoNotSupportedErrorPageWithMalformedUrl() throws Exception {
         List<String> checkMsgs = new ArrayList<String>();
         checkMsgs.add(MessageConstants.MALFORMED_CUSTOM_ERROR_PAGE_CWWKS4317E);
         testHelper.waitForMessages(checkMsgs, true);
+
+        //remove the MALFORMED_URL_ERROR_PAGE from the server config
+        setSpnegoServerConfig(configFile, keytabFile, this.SPN, "false", null, null, null, false);
```

Comment:
we shouldn't need this since the malformedUrl tests use different (the proper) error pages now (notSupportedErrorPage and ntlmErrorPage)

Reply:
You are right, we shouldn't need it. I put it there out of paranoia 😝